client/src/ui/deckbuilder/DeckBuilderMain.vue

```diff
@@ -27,11 +35,13 @@
 <script lang="ts">
 import Vue from 'vue';
 import DeckBuilderSection from './DeckBuilderSection.vue';
+import DeckBuilderSectionControls from "./DeckBuilderSectionControls.vue";
```

Comment:
Nit: use single quotes for all string literals, including import statements (yes I know we haven't been consistent on this so far)